shared/local-debug-live.js

```diff
@@ -1,4 +1,15 @@
-export const dumbFilter = ''
+// @flow
+let envJson = {}
```

Comment:
We could put this logic in a separate module/file since it's exactly the same as in `local-debug.desktop.js`